README.md

```diff
@@ -0,0 +1,2 @@
+# beaker-next-design
+Design and use cases for the new beaker UI
```

Comment:
So, do you wanna use this repository only for new FE of Beaker.Next? Can we get more description here? Based on this info I can't tell if `sketches` dir is a good option.